apps/src/lab2/views/Lab2.tsx

```diff
@@ -39,6 +40,12 @@ const appsProperties: AppProperties[] = [
     backgroundMode: false,
     node: <StandaloneVideo />,
   },
+  {
+    name: 'aichat',
+    usesChannel: false,
```

Comment:
Do we have these values documented somewhere? `usesChannel` indicates a project is channel-backed, yes? I'm not sure about `backgroundMode`.

Reply:
`usesChannel` means that the level is project/channel-backed.  

`backgroundMode` is a new concept that indicates whether the level is removed from the DOM when not active.  For Music Lab, we don't yet support this removal from the DOM due to our Blockly usage not yet supporting a full teardown and subsequent recreation, and so we just make it invisible.  For standalone videos, we do support removal from the DOM.  You can pick based on your lab's capabilities, but it seems most likely you'd want to aim for `false` for this value.